tour/authorizations-code.md

```diff
@@ -41,17 +41,16 @@ static void exercise(AccumuloClient client) throws Exception {
         writer.addMutation(mutation3);
     }
 
-    // Create a second client for the commissioner. Then print all the rows visibile to the
-    // commissioner. Make sure to pass the proper authorizations to the Scanner
-    try (AccumuloClient commishClient = Accumulo.newClient().from(client.properties())
-            .as("commissioner", "gordonrocks").build();
-        Scanner scan = commishClient.createScanner("GothamPD", auths)) {
-        System.out.println("Gotham Police Department Persons of Interest:");
-        for (Map.Entry<Key, Value> entry : scan) {
-            System.out.printf("Key : %-60s  Value : %s\n", entry.getKey(), entry.getValue());
-        }
-    }
-}
+    // Create a second client for the commissioner and output all the rows visible to the commissioner.  
+	// Make sure to pass the proper authorizations.
+	    try (AccumuloClient commishClient = Accumulo.newClient().from(client.properties()).as("commissioner", "gordonrocks").build()){
+	    	Scanner scan = commishClient.createScanner("GothamPD", auths);
+	    	System.out.println("Gotham Police Department Persons of Interest:");
+	    	for (Map.Entry<Key, Value> entry : scan) {
+	            System.out.printf("Key : %-50s  Value : %s\n", entry.getKey(), entry.getValue());
```

Comment:
These are good changes, but it would be nice if they matched our coding style, including using two spaces instead of tab characters. If you are using Eclipse, you can import [our code style](https://github.com/apache/accumulo/blob/main/contrib/Eclipse-Accumulo-Codestyle.xml) to format the code.

Reply:
Thanks for pointing that out. Didn't know we had a specific coding style for this repo until now. I'll change the tab characters to two spaces so the style stays consistent going forward.

Reply:
Yeah, our code is typically formatted to conform to our style by the formatter-maven-plugin when we build/compile, but code on the website isn't built, since it's more documentation than code. So, it doesn't get automatically formatted.

We're not strict about the style in areas where it's not automatically enforced, but it is nice to be somewhat consistent, if we can. Also, I think it's generally understood that we might need to break our style conventions to improve readability (especially in documentation). However, spaces instead of tabs is the one area where we're pretty consistent across all our repos/projects, using them only when necessary (such as in a Makefile).